src/cassio/table/range_operator.py

```diff
@@ -0,0 +1,27 @@
+class RangeOperator:
+    from enum import Enum
+
+    class Operator(Enum):
+        EQ = 1  # Equal
+        LT = 2  # Less Than
+        LTE = 3  # Less Than or Equal to
+        GT = 4  # Greater Than
+        GTE = 5  # Greater Than or Equal to
```

Comment:
chatGPT told me that `!=` is not supported in cassandra, but we have been using it thus far in ColBERT.  

is it newly supported? or is chatGPT just wrong?